packages/p2p/src/components/buy-sell/filter-modal/filter-modal.jsx

```diff
@@ -64,38 +94,7 @@ const FilterModal = () => {
                     <React.Fragment>
                         <FilterModalSearch />
                         <div className='filter-modal__checkbox-container'>
-                            <ThemedScrollbars is_scrollbar_hidden>
-                                {buy_sell_store.is_filter_modal_loading ? (
-                                    <Loading is_fullscreen={false} />
-                                ) : my_profile_store.search_term ? (
-                                    !my_profile_store.search_results || my_profile_store.search_results.length > 0 ? (
-                                        my_profile_store.search_results?.map((payment_method, key) => {
-                                            return (
-                                                <Checkbox
-                                                    key={key}
-                                                    label={payment_method.text}
-                                                    onChange={() => onChange(payment_method)}
-                                                    value={selected_methods.includes(payment_method.value)}
-                                                />
-                                            );
-                                        })
-                                    ) : (
-                                        <FilterModalNoResults text={my_profile_store.search_term} />
-                                    )
-                                ) : (
-                                    my_profile_store.payment_methods_list_items.map((payment_method, key) => {
-                                        return (
-                                            <Checkbox
-                                                name='checkbox'
-                                                key={key}
-                                                label={payment_method.text}
-                                                onChange={() => onChange(payment_method)}
-                                                value={selected_methods.includes(payment_method.value)}
-                                            />
-                                        );
-                                    })
-                                )}
-                            </ThemedScrollbars>
+                            <ThemedScrollbars is_scrollbar_hidden>{getPaymentMethods()}</ThemedScrollbars>
```

Comment:
Since this is a component (it's returning components), it would be like this: 

```suggestion
                            <ThemedScrollbars is_scrollbar_hidden><getPaymentMethods /></ThemedScrollbars>
```

Reply:
done